typings/internalBinding/debug.d.ts

```diff
@@ -0,0 +1,5 @@
+export interface DebugBinding {
+  getV8FastApiCallCount(name: string): number;
+  isEven(value: number): boolean;
+  isOdd(value: number): boolean;
```

Comment:
Can you add a comment to here stating that these functions are enabled only on debug builds? ideally the types should be generated according to the build, but I think it's too impractical to include it inside this pull-request